pkgs/development/python-modules/slicedimage/default.nix

```diff
@@ -38,7 +38,7 @@ buildPythonPackage rec {
 
   nativeCheckInputs = [ pytestCheckHook ];
 
-  # Ignore tests which require setup, check again if disabledTestFiles can be used
```

Comment:
This seems unrelated?

Reply:
It still fixes a typo I noticed while doing this PR